codegen/smithy-ruby-codegen/src/main/java/software/amazon/smithy/ruby/codegen/generators/HttpProtocolTestGenerator.java

```diff
@@ -508,7 +508,8 @@ RubyImportContainer.CGI, getRubyArrayFromList(queryParams))
                     .write("actual_query = $T.parse(request.uri.query)",
                             RubyImportContainer.CGI)
                     .openBlock("expected_query.each do |k, v|")
-                    .write("expect(actual_query[k]).to eq(v)")
+                    .write("actual = actual_query[k].map { |s| s.force_encoding('utf-8') }")
```

Comment:
Not 100% on this one, but looks like query params are only ASCII but protocol tests are UTF-8, so I think no change to our actual builders but instead just compare what we would have sent as utf-8. Doing this in reverse has a frozen string error.